src/lib/clipboard.ts

```diff
@@ -28,9 +28,14 @@ export default class ClipboardHandler {
     return objects;
   };
 
+  /**
+   * Cuts currently selected objects, if any
+   * @return Whether there were objects to cut
```

Comment:
wow are we starting to annotate things now? it feels inconsistent wrt the rest so i would remove this for now and maybe we can add it once we've written more?

Reply:
i mean i'd be okay with removing this since it's only one small annotation, but #116 adds a lot of documentation (although to be fair to more complicated logic) and I'd like to continue to add as much as possible